Peer dependency mongoose@"> 8" causes install failure for version 8.x

When installing version 1.1.23 of @nestjs-cls/transactional-adapter-mongoose with mongoose@8.19.2, npm reports:

>  Could not resolve dependency:
npm error peer mongoose@"> 8" from @nestjs-cls/transactional-adapter-mongoose@1.1.23
npm error node_modules/@nestjs-cls/transactional-adapter-mongoose
npm error   @nestjs-cls/transactional-adapter-mongoose@"^1.1.23" from the root project


To Reproduce

Add "@nestjs-cls/transactional-adapter-mongoose": "1.1.23" and "mongoose": "8.19.2" to dependencies.

Run npm install (without --legacy-peer-deps).

**Observe the ERESOLVE error about peer mongoose@"> 8".**

--- a/packages/transactional-adapters/transactional-adapter-mongoose/package.json
+++ b/packages/transactional-adapters/transactional-adapter-mongoose/package.json
@@ -47,7 +47,7 @@
     },
     "peerDependencies": {
         "@nestjs-cls/transactional": "workspace:^3.1.0",
-        "mongoose": "> 8",
+        "mongoose": ">= 8",
         "nestjs-cls": "workspace:^6.0.1"
     },
     "devDependencies": {

